### PR TITLE
[stable/cerebro] add conditional ingress apiVersion to replace to-be-deprecated extensions/v1beta1

### PR DESCRIPTION
**Pull request by cameronattard**
Signed-off-by: Cameron Attard <cameron.attard@siteminder.com>

#### Is this a new chart

No.

#### What this PR does / why we need it:

Removes usage of to-be-deprecated apiVersion if the newer apiVersion is available.

#### Checklist

- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

@desaintmartin @davidkarlsen @wvidana

### CODE REVIEW

File: stable/cerebro/templates/ingress.yaml [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "cerebro.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion }}
+apiVersion: networking.k8s.io/v1beta1
```
Comment (davidkarlsen) [inline stable/cerebro/templates/ingress.yaml]: Maybe https://github.com/helm/charts/blob/master/stable/anchore-engine/templates/ingress.yaml#L2 ? then we don't need to recollect k8s releases

Comment (cameronattard) [reply stable/cerebro/templates/ingress.yaml]: Much cleaner, thanks. I've amended the PR.

**Pull request merged**